src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -1,15 +1,13 @@
 import angular from 'angular';
 import find from 'lodash/find';
 import every from 'lodash/every';
-import {Observable} from 'rxjs/Observable';
 import 'rxjs/add/operator/mergeMap';
```

Comment:
This still needed?

Reply:
Intelij said it wasn't.  What do you think uses it?

Reply:
I think it was used by a line you deleted. Idk if imports that aren't assigned to a variable get marked correctly in an IDE since it can't check if the variable gets used

Reply:
That's correct.  It was used, but I removed that code.